docs/source/en/model_doc/gptj.md

```diff
@@ -15,119 +15,97 @@ rendered properly in your Markdown viewer.
-... )
-
->>> input_ids = tokenizer(prompt, return_tensors="pt").input_ids
-
->>> gen_tokens = model.generate(
-...     input_ids,
-...     do_sample=True,
-...     temperature=0.9,
-...     max_length=100,
-... )
->>> gen_text = tokenizer.batch_decode(gen_tokens)[0]
+pipeline = pipeline(
+    task="text-generation",
+    model="EleutherAI/gpt-j-6B",
+    torch_dtype=torch.float16,
+    device=0
+)
+pipeline(
+    "In a shocking finding, scientists discovered a herd of unicorns living in a remote, previously unexplored valley, in the Andes Mountains."
+)
 ```
 
-...or in float16 precision:
+</hfoption>
+<hfoption id="AutoModel">
 
-```python
->>> from transformers import GPTJForCausalLM, AutoTokenizer
->>> import torch
+```py
+from transformers import GPTJForCausalLM, AutoTokenizer
+import torch
 
->>> device = "cuda"
->>> model = GPTJForCausalLM.from_pretrained("EleutherAI/gpt-j-6B", torch_dtype=torch.float16).to(device)
->>> tokenizer = AutoTokenizer.from_pretrained("EleutherAI/gpt-j-6B")
+device = "cuda"
+model = GPTJForCausalLM.from_pretrained("EleutherAI/gpt-j-6B", torch_dtype=torch.float16).to(device)
+tokenizer = AutoTokenizer.from_pretrained("EleutherAI/gpt-j-6B")
 
->>> prompt = (
-...     "In a shocking finding, scientists discovered a herd of unicorns living in a remote, "
-...     "previously unexplored valley, in the Andes Mountains. Even more surprising to the "
-...     "researchers was the fact that the unicorns spoke perfect English."
-... )
+prompt = (
+    "In a shocking finding, scientists discovered a herd of unicorns living in a remote, "
+    "previously unexplored valley, in the Andes Mountains. Even more surprising to the "
+    "researchers was the fact that the unicorns spoke perfect English."
+)
 
->>> input_ids = tokenizer(prompt, return_tensors="pt").input_ids.to(device)
+input_ids = tokenizer(prompt, return_tensors="pt").input_ids.to(device)
 
->>> gen_tokens = model.generate(
-...     input_ids,
-...     do_sample=True,
-...     temperature=0.9,
-...     max_length=100,
-... )
->>> gen_text = tokenizer.batch_decode(gen_tokens)[0]
+gen_tokens = model.generate(
+    input_ids,
+    do_sample=True,
+    temperature=0.9,
+    max_length=100,
+)
+gen_text = tokenizer.batch_decode(gen_tokens)[0]
 ```
 
+</hfoption>
+<hfoption id="transformers CLI">
+
+```bash
+echo -e "In a shocking finding, scientists discovered a herd of unicorns living in a remote, previously unexplored valley, in the Andes Mountains." | transformers run 
+    --task text-generation --model EleutherAI/gpt-j-6B --device 0
+```
+
+</hfoption>
+</hfoptions>
+
+## Notes
+
+- GPT-J-6B is not intended for deployment without fine-tuning, supervision, and/or moderation. It may generate harmful or offensive text and is not suitable for human-facing interactions.
+
+- GPT-J-6B has not been fine-tuned for downstream contexts such as writing genre prose or commercial chatbots. It does not respond to prompts like ChatGPT does.
```

Comment:
```suggestion
- Training requires at least 4x model size GPU memory even with mixed precision. Explore options such as DeepSpeed or use the original codebase to train and fine-tune the model on TPU and convert to Transformers for inference (see [here](https://github.com/kingoflolz/mesh-transformer-jax/blob/master/howto_finetune.md)).

- Although the embedding matrix is 50400, only 50257 entries are used. The extra tokens are added for TPU efficiency. To avoid a mismatch between embedding matrix size and vocab size, the GPT-J tokenizer contains 143 extra tokens (`<|extratoken_1|>... <|extratoken_143|>`.
```